src/cosmicqc/analyze.py

```diff
@@ -138,6 +144,10 @@ def find_outliers(
             Outlier data frame for the given conditions.
     """
 
+    # interpret the df as SCDataFrame
```

Comment:
Sorry to come late to the party, but what is the difference between `find_outliers()` and `identify_outliers()`?

I make this comment noticing a pattern of duplicate comments (i wanted to make the same suggested comment as to why SCDataframe here too)

Reply:
No worries, and thank you for the good question. `find_outliers()` prints output and shows the outliers along with potential metadata if included. `identify_outliers()` creates a mask for a single threshold set as True or False or provides that mask along with the zscores associated with the mask for further analysis.

On SCDataFrame, similarly, I hope we can use methods from the class to help with CLI efforts through this function.